colin-api/src/colin_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.123.0'  # pylint: disable=invalid-name
+__version__ = '2.123.1'  # pylint: disable=invalid-name
```

Comment:
No need to update version in each pr (in lear repo). This will be updated together with other apps (in lear repo) before each release.